postgresql/Dockerfile

```diff
@@ -16,7 +16,7 @@
 
 # https://hub.docker.com/r/bitnami/postgresql/tags/
 # https://quay.io/repository/bitnami/postgresql?tab=tags
-FROM bitnami/postgresql:11.15.0-debian-10-r76
+FROM bitnami/postgresql:11.15.0-debian-10-r93
```

Comment:
Should we bump this to postgres 13 or 14? Seems like we should, especially since we're planning to run 14 in prod soon.

Also that quay.io comment is invalid. They seem to have deleted their Quay.io account.

Reply:
Yes, lets get it up to postgres 14.